recipes/bzip2/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: bzip2
+  version: 1.0.6
+
+source:
+  fn: bzip2-1.0.6.tar.gz
+  url: http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz
```

Comment:
Please template out the versions. Take a look at my [PR]( https://github.com/conda-forge/staged-recipes/pull/822/files#diff-245f63c98c2671f2913ba880ae7d9a25R1 ) to see an example of what this would look like.